lib/rubocop/cop/style/empty_else.rb

```diff
@@ -21,25 +82,33 @@ def on_case(node)
         private
 
         def check(node, else_clause)
-          return unless node.loc.else
-          return if else_clause && else_clause.type != :nil
+          case style
+          when :empty
+            empty_check(node, else_clause)
+          when :nil
+            nil_check(node, else_clause)
+          when :both
+            both_check(node, else_clause)
+          end
+        end
 
-          add_offense(node, :else, MSG)
+        def empty_check(node, else_clause)
+          add_offense(node, :else, MSG) if node.loc.else && else_clause.nil?
         end
 
-        def if_else_clause(node)
-          keyword = node.loc.keyword
-          if keyword.is?('if')
-            node.children[2]
-          elsif keyword.is?('elsif')
-            node.children[2]
-          elsif keyword.is?('unless')
-            node.children[1]
-          end
+        def nil_check(node, else_clause)
+          add_offense(node, node.location, MSG) if else_clause &&
```

Comment:
RuboCop complains about `nil_check` being written as a regular `if`.

When written as
```ruby
def nil_check(node, else_clause)
  if else_clause && else_clause.type == :nil
    add_offense(node, node.location, MSG)
  end 
end 
```
RuboCop outputs:
```shell
lib/rubocop/cop/style/empty_else.rb:100:11: C: Use a guard clause instead of wrapping the code inside a conditional expression.
          if else_clause && else_clause.type == :nil
```

What is your suggestion for this?